src/components/UpdateChecker.tsx

```diff
@@ -0,0 +1,60 @@
+import { useInfoToast } from '@/lib/customToast';
+import { useEffect } from 'react';
+
+const UpdateChecker = () => {
+  const currentVersion = process.env.CONFIG_CURRENT_VERSION || '0.0.0';
+  const infoToast = useInfoToast();
+
+  useEffect(() => {
+    fetch('https://raw.githubusercontent.com/Cyronlee/zBoard/main/package.json')
+      .then((res) => res.json())
+      .then((data) => {
+        const latestVersion = data.version;
+        if (compareVersions(latestVersion, currentVersion)) {
+          infoToast('New update available', 'Please update.');
+        }
+      })
+      .catch(() => console.warn('Fetch latest version failed.'));
+  }, []);
+  return <></>;
+};
+
+const compareVersions = (v1: string, v2: string) => {
```

Comment:
can we just compare string like '1.13.0' > '1.9.0'

Reply:
> '1.1'>'1.90'
> -> false